src/display.js

```diff
@@ -1,7 +1,24 @@
 export default class Fetch {
-     static displayMeals = async (category = 'Beef') => {
-       const mealsList = await fetch(`https://themealdb.com/api/json/v1/1/filter.php?c=${category}`);
-       const response = await mealsList.json();
-       return response;
-     }
+  static async displayMeals(category = 'Breakfast') {
+    const mealsList = await fetch(`https://themealdb.com/api/json/v1/1/filter.php?c=${category}`);
+    const response = await mealsList.json();
+    return response;
+  }
+
+  static async addLike(meal) {
+    const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Kw76R2UqPqv86GuooSFn/likes/', {
```

Comment:
The local variable response is redundant. You can add an inline return directly here No need to use a variable in this case

Reply:
- [ ] I think It will be better if you create a global variable where you can store the part of your URL that doesn't change so that you avoid repeating yourself. You may have a hard time when it comes to updating your URL key since you have to look for the URL link everywhere in your code but having it stored in a variable will help you change it easily